__tests__/app.test.js

```diff
@@ -0,0 +1,39 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const testData = require("../db/data/test-data");
+
+beforeEach(() => seed(testData));
+
+afterAll(() => db.end());
+
+describe("GET /api/topics", () => {
+  test("status:200, responds with an Array has 3 objects topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Array);
```

Comment:
We are expecting the response to be an object, with a key and a value of an array. Have a look at your endpoints.json for an idea of what I mean.